UpdateProfile.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  </head>
+  <body>
+    <div class="container">
+      <div class="post-form">
+        <div class="form-row mb-lg-4">
+          	<img src="Jaime.jpg" class="rounded-circle" style="height:200px; margin-left:15px; padding: 10px;" />
+
+            <div class="form-column">
+              <div class="form-group form-inline">
+                <label for="exampleInputEmail1" class="col-lg-3 col-form-label" style="padding:10px">Name:</label>
+                <input type="text" class="form-control" id="Name" placeholder="Name">
+              </div>
+
+              <div class="form-group form-inline">
+                <label for="exampleInputEmail1" class="col-lg-3 col-form-label" style="padding:10px" >Contact Number:</label>
+                <input type="text" class="form-control" id="exampleInputEmail1" placeholder="0917XXXXX">
+              </div>
+            </div>
```

Comment:
Good so far, but spacing can be improved on further. Suggestions:
1. Change `form-group form-inline` to `form-row` for easier control with the spacing of the label and the input.
2. Increase the space you allotted for the labels, so that the Contact Number label won't have to take two lines. Increase it maybe to `col-lg-5` instead of 3. Remember that in a form row, the numbers should add up to at most 12, so if the label is `col-lg-5`, the other tags in that row should sum up to 7.
3. To be able to have control over the length of the input field, wrap the <input> tag with a div then add `col-lg-x` where x is the space allotted.